### PR TITLE
Separate routes

### PR DESCRIPTION
**Pull request by MariaTuruta**

### CODE REVIEW

File: src/routes/courses-route.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+const express = require('express');
+const errors = require('@feathersjs/errors');
+
+module.exports = function createCoursesRoute(connection) {
+  const repository = require('../repositories/courses-repository')(connection.models); // eslint-disable-line global-require
+  const router = express.Router();
+
+  router.route('/')
+    .get(list) // eslint-disable-line no-use-before-define
```
Comment (alxolr) [inline src/routes/courses-route.js]: Put `no-use-before-define` in off in eslint rules

File: src/repositories/admins-repository.js [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+  function list(queryParams) {
+    const { limit, offset, contains } = queryParams;
+
+    const getData = Admin.findAll({
+      offset,
+      limit,
+      where: { firstName: { [Op.like]: [`%${contains}%`] } },
+    });
+
+    const getTotal = Admin.findAndCountAll();
```
Comment (ramzes13) [inline src/repositories/admins-repository.js]: http://docs.sequelizejs.com/manual/tutorial/models-usage.html#-findandcountall-search-for-multiple-elements-in-the-database-returns-both-data-and-total-count

you do not need the first request, `findAndCountAll` return rows and total :)

File: src/routes/courses-route.js [hunk 3]
```diff
@@ -2,36 +2,44 @@
...
       })
       .catch(next);
   }
 
   function list(request, response, next) {
-    repository.list()
-      .then((result) => {
-        response.json(result);
+    repository.list(request.query)
+      .then(([data, total]) => {
```
Comment (ramzes13) [inline src/routes/courses-route.js]: you can create a middleware with will generate this response

File: src/repositories/admins-repository.js (hunk 2 above)
Comment (MariaTuruta) [reply src/repositories/admins-repository.js]: first request is for selected result with limits and offset from database, and the second is for counting all the rows in database, despite the selection..

**Pull request merged**